Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.3-tentative.plan

```diff
@@ -0,0 +1,252 @@
+            {"path": "tests/bouncing-particles/resources/bouncing-canvas-particles.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-canvas-shapes.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-css-images.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-css-shapes.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-particles.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-svg-images.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-svg-particles.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-svg-shapes.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/bouncing-tagged-images.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/image1.jpg", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/image2.jpg", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/image3.jpg", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/image4.jpg", "mode": "100644", "type": "blob"},
+            {"path": "tests/bouncing-particles/resources/image5.jpg", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom", "mode": "040000", "type": "tree"},
+            {"path": "tests/dom/compositing-transforms.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/focus.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/leaves.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/multiply.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/particles.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/resources", "mode": "040000", "type": "tree"},
+            {"path": "tests/dom/resources/compositing-transforms.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/resources/dom-particles.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/resources/focus.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/resources/leaves.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/dom/resources/multiply.js", "mode": "100644", "type": "blob"},
+            {"path": "tests/core", "mode": "040000", "type": "tree"},
+            {"path": "tests/core/canvas-stage.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/core/design.html", "mode": "100644", "type": "blob"},
+            {"path": "tests/core/focus.html", "mode": "100644", "type": "blob"},
```

Comment:
This was removed. Per discussion, `github_subtree` needs to be updated accordingly.
The way to get this updated accordingly is:
1. Remove existing `github_subtree` in `Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.3-tentative.plan`
2. Run `python3 Tools/Scripts/webkitpy/benchmark_runner/plan_file_optimizer.py Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.3-tentative.plan`